### PR TITLE
No fatal flag

### PR DESCRIPTION
**Pull request by Daanikus**
Allows us to proceed after error like so:

```
# dan @ dan-ThinkPad-T580 in ~/code/action-test on git:master x [16:07:41] 
$ lingo run review --debug --lingo-file test.yaml           
2019-08-19T16:08:32.910+1200	DEBUG	review/cli.go:98	reviewCMD called
2019-08-19T16:08:32.910+1200	DEBUG	review/cli.go:108	insecure false
Syncing your repo...
Running review flow...
2019-08-19T16:08:34.160+1200	DEBUG	review/review.go:27	sending request to flow server...
2019-08-19T16:08:34.160+1200	DEBUG	flow/request.go:44	opening connection to flow server ...
2019-08-19T16:08:34.160+1200	DEBUG	service/service.go:75	getting tls cert from cache...
2019-08-19T16:08:34.160+1200	DEBUG	service/service.go:129	dialing grpc server...
2019-08-19T16:08:34.160+1200	DEBUG	service/service.go:108	...got answer from grpc server.
2019-08-19T16:08:34.160+1200	DEBUG	flow/request.go:49	...connection to flow server opened
2019-08-19T16:08:34.688+1200	DEBUG	review/review.go:34	...request to flow server sent. Received reply channel.
2019-08-19T16:08:35.166+1200	DEBUG	flow/cli.go:258	Result error: /home/dan/go/src/github.com/codelingo/codelingo/sdk/flow/request.go:73: flow config: codelingo.yaml file 'codelingo.yaml': tenet 'declare-empty-slice' must have a review config section
2019-08-19T16:08:35.166+1200	DEBUG	flow/cli.go:221	/home/dan/go/src/github.com/codelingo/codelingo/sdk/flow/request.go:73: flow config: codelingo.yaml file 'codelingo.yaml': tenet 'declare-empty-slice' must have a review config section
/home/dan/go/src/github.com/codelingo/codelingo/sdk/flow/cli.go:260: 
flow config: codelingo.yaml file 'codelingo.yaml': tenet 'declare-empty-slice' must have a review config section
exit status 1

# dan @ dan-ThinkPad-T580 in ~/code/action-test on git:master x [16:07:07] 
$ lingo run review --debug --lingo-file test.yaml --no-fatal
WARNING: command running in 'no fatal error' mode
2019-08-19T16:07:36.731+1200	DEBUG	review/cli.go:98	reviewCMD called
2019-08-19T16:07:36.731+1200	DEBUG	review/cli.go:108	insecure false
Syncing your repo...
Running review flow...
2019-08-19T16:07:38.170+1200	DEBUG	review/review.go:27	sending request to flow server...
2019-08-19T16:07:38.170+1200	DEBUG	flow/request.go:44	opening connection to flow server ...
2019-08-19T16:07:38.170+1200	DEBUG	service/service.go:75	getting tls cert from cache...
2019-08-19T16:07:38.170+1200	DEBUG	service/service.go:129	dialing grpc server...
2019-08-19T16:07:38.170+1200	DEBUG	service/service.go:108	...got answer from grpc server.
2019-08-19T16:07:38.170+1200	DEBUG	flow/request.go:49	...connection to flow server opened
2019-08-19T16:07:38.972+1200	DEBUG	review/review.go:34	...request to flow server sent. Received reply channel.
2019-08-19T16:07:39.669+1200	DEBUG	flow/cli.go:258	Result error: /home/dan/go/src/github.com/codelingo/codelingo/sdk/flow/request.go:73: flow config: codelingo.yaml file 'codelingo.yaml': tenet 'declare-empty-slice' must have a review config section
2019-08-19T16:07:41.109+1200	DEBUG	flow/request.go:77	got reply %vpayload:<type_url:"/flow.Issue" value:"\n\035comment-first-word-when-empty\022#\n\017\n\007main.go\020j\030\r \001\022\020\n\007main.go\020\233\001\030\020 \002\032\343\001Every exported function in a program should have a doc comment. The first sentence should be a summary that starts with the name (Thing) being declared.\nFrom [effective go](https://golang.org/doc/effective_go.html#commentary).\n\"\002}\n*1func Thing() {\n\tt := []string{}\n\tfmt.Println(t)\n}2,\nfunc foo(msg string, ctx context.Context) {P\001\202\001\016review comment" > 
main.go:16:2

    Every exported function in a program should have a doc comment. The first sentence should be a summary that starts with the name (Thing) being declared.
    From [effective go](https://golang.org/doc/effective_go.html#commentary).
    

    ...
    }
    
  + func Thing() {
  + 	t := []string{}
  + 	fmt.Println(t)
  + }
    
    func foo(msg string, ctx context.Context) {
    ...

[o]pen [k]eep [d]iscard : d


 ---------------
NO FATAL MODE: The following 1 errors were ignored:
ERROR: flow config: codelingo.yaml file 'codelingo.yaml': tenet 'declare-empty-slice' must have a review config section
---------------

Done! No issues found.
```

### CODE REVIEW

File: sdk/flow/cli.go [hunk 1]
```diff
@@ -178,6 +178,11 @@ func setBaseApp(cliApp *CLIApp) {
 			Usage: "the Tenet `FILE` to run rewrite over. If the flag is not set, codelingo.yaml files are read from the branch being rewritten.",
 			//	Destination: &language,
 		},
+		cli.BoolFlag{
+			Name:  "no-fatal",
+			Usage: "the command will be run in 'no-fatal' mode, which will allow an Action to keep running even after receiving an error",
+			//	Destination: &language,
```
Comment (BlakeMScurr) [inline sdk/flow/cli.go]:
```suggestion
```

Needed?

File: sdk/flow/cli.go [hunk 2]
```diff
@@ -306,6 +320,15 @@ l:
 		}
 	}
 
+	if isNoFatalMode {
+		fmt.Println("\n\n", strings.Repeat("-", 15))
+		fmt.Printf("NO FATAL MODE: The following %d errors were ignored:\n", len(errs))
+		for _, err := range errs {
+			fmt.Println("ERROR:", err.Error())
```
Comment (BlakeMScurr) [inline sdk/flow/cli.go]:
```suggestion
			fmt.Println("ERROR:", errors.ErrorStack(err))
```

File: sdk/flow/cli.go (hunk 1 above)
Comment (Daanikus) [reply sdk/flow/cli.go]: @BlakeMScurr Whoops, left over from copy/paste

**Comment by BlakeMScurr** [comment]
We should also add an `errorsOutput` flag to capture the errors from a file. That can be for later.

**Pull request merged**